base_station/gui/src/components/WaypointEditor.vue

```diff
@@ -128,6 +132,16 @@ export default {
 
     this.$parent.subscribe('/nav_status', (msg) => {
       this.nav_status = msg
+      if(this.waitingForNav){
+        if(this.nav_status.nav_state_name!="Off" && this.autonEnabled){
+          this.waitingForNav = false
+          this.autonButtonColor = "green"
+        }
+        else if(this.nav_status.nav_state_name=="Off" && !this.autonEnabled){
```

Comment:
@zwgold have you fixed the bug where nav doesn't go to off when auton is disabled?

Reply:
@cgiger00 Refer to #1004 for the fix to make sure we go to off if auton is disabled